src/AiSystem.cpp

```diff
@@ -18,60 +23,26 @@ AiSystem::AiSystem()
 
 void AiSystem::update() {
     // only update every 10 frames
-    if (++counter_ == 10) {
+    if (++counter_ == 120) {
         counter_ = 0;
     }
 
     for (int i = 3; i >= 4 - num_ai_; i--) {
-        if (i != whos_it_) {
-            path_to(i, cars_[whos_it_].position_);
-        } else if (counter_ == 0) {
-            unsigned int random = rand() % 20;
-
-            if (random == 0) {
-                EventSystem::queue_event(
-                    Event(
-                        EventType::KEYPRESS_EVENT,
-                        "player_id", i,
-                        "key", SDL_CONTROLLER_AXIS_LEFTX,
-                        "value", -32768
-                    )
-                );
-            } else if (random == 1) {
-                EventSystem::queue_event(
-                    Event(
-                        EventType::KEYPRESS_EVENT,
-                        "player_id", i,
-                        "key", SDL_CONTROLLER_AXIS_LEFTX,
-                        "value", 32768
-                    )
-                );
-            } else {
-                EventSystem::queue_event(
-                    Event(
-                        EventType::KEYPRESS_EVENT,
-                        "player_id", i,
-                        "key", SDL_CONTROLLER_AXIS_TRIGGERRIGHT,
-                        "value", 32768
-                    )
-                );
-            }
-
-            random = rand() % 40;
-
-            if (random < 2) {
-                EventSystem::queue_event(
-                    Event(
-                        EventType::KEYPRESS_EVENT,
-                        "player_id", i,
-                        "key", random == 0 ? SDL_CONTROLLER_BUTTON_LEFTSHOULDER : SDL_CONTROLLER_BUTTON_RIGHTSHOULDER,
-                        "value", SDL_CONTROLLERBUTTONDOWN
-                    )
-                );
-            }
+        choose_ai_state(i);
```

Comment:
This function does alot more than choose the ai state IMO. I think so that the names make sense, there should be no `choose_ai_state()` func, and the code should just go here in the update.